include/aspect/material_model/rheology/diffusion_dislocation.h

```diff
@@ -90,6 +90,33 @@ namespace aspect
                                             const double temperature,
                                             const SymmetricTensor<2,dim> &strain_rate) const;
 
+          /**
+           * Compute the derivative of the natural logarithm of the strain rate
+           * with respect to the natural logarithm of the stress
+           * at the current evaluation point (log stress) estimated by KINSOL.
```

Comment:
Drop the reference to KINSOL here. The documentation should describe *what* a function computes, not *how* it computes that.